hedera-node/docs/design/services/smart-contract-service/cancun-fork-support.md

```diff
@@ -0,0 +1,91 @@
+# Cancun Fork Support
+
+## Purpose
+
+There Ethereum "Cancun" hard introduces a number of changes to the EVM that will need to be implemented to maintain EVM
+Equivalence. There are four different HIPs covering this feature (HIP-865: Add EVM Support for transient storage and
+memory copy Cancun opcodes, HIP-866: Add EVM compatibility for non-supported Cancun blob features, HIP-867: Add Smart
+Contract Services Support for KZG Point Evaluation Precompiled Function, and HIP-868: Support Cancun Self-Destruct
+Semantics in Smart Contract Services //TODO link hips once numbers are assigned and published). This document describes
+how the HIPs will be implemented in Hedera Services.
+
+Generally speaking there are four strategies that will be adopted:
+
+1. Adopt without change. This includes the `MCOPY`, `TSTORE`, and `TLOAD` operations as well as the KZG precompile.
+2. Adapt with Hedera Considerations. This includes the `SELFDESTRUCT` changes.
+3. Do not adopt but stub out placeholders. This includes the `VERSIONEDHASH` and `BLOBBASEFEE` operations.
+4. Do not adopt. This includes all Blob handling and the "Type 3" blob transaction.
+
+### Example User Story
+
+* As a smart contract developer, I want to use current versions of solidity that may generate opcode that are only
+  available in the Cancun fork.
+* As a Hedera developer, I want to preserve maximum furuture design space to adopt, adopt, or not adopt blobs.
+* As an end user, I want prompt and accurate faiures if I attempt to use Blob features in Hedera.
+
+## Goals
+
+The principal goal is to maintain EVM and Solidity/Vyper/etc. compatibility, where the bytecode from contracts compiled
+with Cancun enabled smart contract compilers still function correctly.
+
+Where inclusion of features don't requre additional work, they too should be included.
+
+Where we have existing overrides, opcodes that change those behaviors should still operate in a fashion that is in
+harmony with the EVM equivalence requirements and the Hedera requirements to the extent possible. Working with Hedera
+takes priority when there is unavoidable conflict.
+
+Opcodes that interact with blobs should not cause the contracts to break, but should not require blobs to work.
+
+## Non Goals
+
+It is not a goal to introduce blobs into Hedera. It is also not a goal to restrict design space nor to dictate future
+design directions for support or non-support of blobs.
+
+## Implementation
+
+### New Cancun EVM
+
+A new EVM version will be created that will be based off of the Cancun EVM. Simply activating the Cancun EVM should get
```

Comment:
Cna we be explicit on what activating means? E.g. what config needs to be xhanged and to what?